packages/sync-actions/test/order-sync.spec.js

```diff
@@ -21,64 +21,321 @@ describe('Actions', () => {
     orderSync = orderSyncFn()
   })
 
-  test('should build *state actions', () => {
-    const before = {
-      orderState: 'Open',
-      paymentState: 'Pending',
-      shipmentState: 'Ready',
-    }
-    const now = {
-      orderState: 'Complete',
-      paymentState: 'Paid',
-      shipmentState: 'Shipped',
-    }
-
-    const actual = orderSync.buildActions(now, before)
-    const expected = [
-      { action: 'changeOrderState', orderState: 'Complete' },
-      { action: 'changePaymentState', paymentState: 'Paid' },
-      { action: 'changeShipmentState', shipmentState: 'Shipped' },
-    ]
-    expect(actual).toEqual(expected)
+  describe('base', () => {
+    test('should build *state actions', () => {
+      const before = {
+        orderState: 'Open',
+        paymentState: 'Pending',
+        shipmentState: 'Ready',
+      }
+      const now = {
+        orderState: 'Complete',
+        paymentState: 'Paid',
+        shipmentState: 'Shipped',
+      }
+
+      const actual = orderSync.buildActions(now, before)
+      const expected = [
+        { action: 'changeOrderState', orderState: 'Complete' },
+        { action: 'changePaymentState', paymentState: 'Paid' },
+        { action: 'changeShipmentState', shipmentState: 'Shipped' },
+      ]
+      expect(actual).toEqual(expected)
+    })
   })
 
-  test('should build `addDelivery` action', () => {
-    const before = {
-      shippingInfo: {
-        deliveries: [],
-      },
-    }
-    const now = {
-      shippingInfo: {
-        deliveries: [
-          {
-            items: [{ id: 'li-1', qty: 1 }, { id: 'li-2', qty: 2 }],
-            parcels: [
-              {
-                measurements: {
-                  heightInMillimeter: 1,
-                  lengthInMillimeter: 1,
-                  widthInMillimeter: 1,
-                  weightInGram: 1,
-                },
-                trackingData: {
-                  trackingId: '111',
+  describe('deliveries', () => {
+    test('should build `addDelivery` action', () => {
+      const before = {
+        shippingInfo: {
+          deliveries: [],
+        },
+      }
+      const now = {
+        shippingInfo: {
+          deliveries: [
+            {
+              items: [{ id: 'li-1', qty: 1 }, { id: 'li-2', qty: 2 }],
+              parcels: [
```

Comment:
`deliveries`, `parcels` and `address` are all optional for the `addDelivery` action, and I would suggest that the test should reflect that, by breaking the test(s) into testing `parcels`, `deliveries` and `address` separately.


ref: https://docs.commercetools.com/http-api-projects-orders.html#add-delivery

Reply:
This is not mine, I think is marked as new because of auto-formatting. Do you want me to look into it?

Reply:
aaaah... I seee.
nevermind then :)